src/components/Modals/TourWindow.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <div class="tour">
+    <div class = "blackout">
+      <div class ="intropage">
+        <div class = "picture">
+          <img src = "@/assets/images/Person_planning.svg" alt = "person planning">
+        </div>
+        <div class = "content">
+          <div class = "title">
+            {{ title }}
+          </div>
+          <div class = "body">
+            {{ text }}
+          </div>
+          <button v-on:click = "$emit('hide'); $emit('startTour')">
```

Comment:
Consider using the shorthand here and elsewhere in the file. Also consider making a function to call (but can go either way)

Reply:
Bringing this old comment back :) `v-on:click` can just be `@click` here and throughout this file